CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Upgrade MSAL from `4.55.0` to `4.59.1`.
+
```

Comment:
I'm not sure how granular we typically go with our changelog, but do we list general security improvements and bug fixes? I ask because we did make some additional changes yesterday to patch some vulnerabilities tied to specific dependencies.

Reply:
That's a great question. So, we usually only mention when some major dependency changes (you will be able to see similar mentions in past in our changelog). MSAL happens to be the only one in our case. Other than that, we usually mention user facing changes (even if it's a very small change).

Reply:
Sounds good to me!